browser/ui/views/tabs/brave_tab_container.cc

```diff
@@ -110,14 +111,14 @@ base::OnceClosure BraveTabContainer::LockLayout() {
 
 void BraveTabContainer::AddedToWidget() {
   TabContainerImpl::AddedToWidget();
-  auto* browser = tab_slot_controller_->GetBrowser();
+  auto* browser = const_cast<Browser*>(tab_slot_controller_->GetBrowser());
```

Comment:
why are you using `const_cast` here? I need to check the semgrep rules because this should be flagged cc @thypon

Reply:
Looks like maybe I never added these, I'll create a PR for it

Reply:
I think you could case `controller_` -> `TabStrip` which has `GetBrowserWindowInterface`

Reply:
It works! Done.